Makefile

```diff
@@ -72,6 +72,11 @@ install: build
 parser:
 	make -C go/vt/sqlparser
 
+visitor:
```

Comment:
shouldn't we be calling this whenever there is a change to sql.y (and sql.go)?

Reply:
not really. only when there is a change to `ast.go`

Reply:
will people need to know to call this manually? is there a test that will catch it if someone forgets to do so after changing ast.go?

Reply:
added pre-commit hook